shared/models.go

```diff
@@ -89,7 +89,7 @@ type RulePolicy struct {
 	Method  string `yaml:"method"`
 	Key     string `yaml:"key"`
 	Value   string `yaml:"value"`
-	Latency int64  `yaml:"latency"`
+	ResponseTime int64  `yaml:"response-time"`
```

Comment:
formatting, this struct: 
type RulePolicy struct {
	Type    string `yaml:"type"`
	Service string `yaml:"service"`
	Path    string `yaml:"path"`
	Method  string `yaml:"method"`
	Key     string `yaml:"key"`
	Value   string `yaml:"value"`
	ResponseTime int64  `yaml:"response-time"`
	Name    string `yaml:"name"`
}

should be formatted like this:

```
type RulePolicy struct {
	Type         string `yaml:"type"`
	Service      string `yaml:"service"`
	Path         string `yaml:"path"`
	Method       string `yaml:"method"`
	Key          string `yaml:"key"`
	Value        string `yaml:"value"`
	ResponseTime int64  `yaml:"response-time"`
	Name         string `yaml:"name"`
}
```

(types and annotations of the properties start from the same place)

Reply:
Sorry about that, i'm formatting manually, forgot about it.